tests/unit/lms/views/admin/application_instance_test.py

```diff
@@ -354,11 +354,14 @@ def test_update_instance(
         )
 
     def test_update_application_instance(
-        self, pyramid_request, application_instance_service, views
+        self,
+        views,
+        pyramid_request,
+        application_instance_service,
     ):
-        pyramid_request.params = {
+        pyramid_request.params = pyramid_request.POST = {
             "name": "  NAME  ",
-            "lms_url": "   http://example.com    ",
+            "lms_url": "http://example.com",
```

Comment:
By using the URL pattern we no longer accept white space (it will say it's not a valid URL)